### ISSUE TITLE
[Screen Reader - Power Apps Admin - Scheduling] In scan mode,Screen Reader does not announce the disable state of combo box under Scheduling tab item

### ISSUE THREAD

**Issue by sonimishra** [OfficeDev/office-ui-fabric-react#7102]
<!-- 🚨 STOP 🚨 𝗦𝗧𝗢𝗣 🚨 𝑺𝑻𝑶𝑷 🚨

We’re here to help, but we need a few things from you! 

  Please provide a reproduction of the bug in a codepen, if possible. Here’s how:
 User Experience:
If, the state is not being announced , then user who depends upon screen reader for navigation might get affected and as he does not get the complete information about the control.

MAS Ref:
MAS 4.1.2 - Name, Role, Value (40B)

Keros statement : Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.  
  
### Environment Information 
 Test Environment:
OS version: Version 1803 (OS build 17134.167)
Browser: Edge, Chrome
Screen Reader: Narrator, JAWS

- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: 

### Please provide a reproduction of the bug in a codepen:
 Pre-Requisite:
1. Turn on narrator by pressing Ctrl + window + ENTER combination.

Repro Steps:
1. Navigate to https://admin.powerplatform.microsoft.com/ext/DataIntegration and login with intusr.
2. Navigate to the Data Integration menu item on the left bar and select it.
3. Navigate on the controls of data integration page thus opened.
4. Navigate to 'Projects' tab and select it . 
5. Navigate to any already existing project and select it. Project edit screen will open.
6. Navigate to 'Scheduling' tab and select it and turn on scan mode (Caps+Space bar).
7. Navigate to 'Frequency unit' control present under 'Connections' tab
8. Verify Screen Reader does not announce the disable state of 'Frequency unit' combo box.
 
#### Actual behavior:
 Screen Reader does not announce the disable state of 'Frequency unit' combo box. It announce it as 'Frequency unit, Selected, selection contains 0 item, hours.
 
#### Expected behavior:
 Screen Reader should announce the disable state of 'Frequency unit' combo box. It should announce it as 'Frequency unit' combo box disable.

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) : Blocking
 
Products/sites affected: (if applicable)

**Comment by aneeshack4** [comment]
@sonimishra please fill out the bug template & provide information such as which Fabric component this bug's being reported on, screenshots, repro steps, more details on expected vs. current behavior - always do this when you're filing issues on Fabric. 
Right now, from your description, all I could do was go to the combo box control on the Fabric components site & run Narrator (you didn't specify which screen reader to use either so if it's not Narrator please let me know). I ran it on scan mode and couldn't repro: 
![image](https://user-images.githubusercontent.com/4161791/48506349-18e26600-e7fe-11e8-8d21-928666990243.png)
